### PR TITLE
Update playground with new syntax

### PR DESCRIPTION
**Pull request by arukshani**
$subject

### CODE REVIEW

**Review by Bhashinee — APPROVED** [review]

**Comment by riyafa** [comment]
shall we have a custom method to handle errors from the respond method?

**Comment by wggihan** [comment]
Shall we remove all the unnecessary empty lines between statements?

**Comment by arukshani** [comment]
If you look at the bio page playground widget you will see that the empty lines have been kept for readability. So IMO we should keep them as it is doing it otherwise will make the code look jammed

**Review by wggihan — APPROVED** [review]

File: src/resilient_service.bal [hunk 1]
```diff
@@ -1,94 +1,107 @@
...
         errResponse.statusCode = http:OK_200;
-        _ = caller -> respond(errResponse);
-      }
+        var result = caller->respond(errResponse);
+        handleError(result);
     }
   }
 }
+
+function handleError(error? result) {
```
Comment (TylerJewell) [inline src/resilient_service.bal]: don't include abstracted functions in the playground samples - it makes it harder to follow in a limited view playpen. This method is only called once so just inline it for readability.

Comment (arukshani) [reply src/resilient_service.bal]: Simplified the code and issued a new PR >> https://github.com/ballerina-guides/playground-circuit-breaker/pull/28/files

**Pull request merged**